masonry/src/tracing_backend.rs

```diff
@@ -99,10 +101,10 @@ pub(crate) fn try_init_layered_tracing() -> Result<(), SetGlobalDefaultError> {
     Ok(())
 }
 
-pub(crate) fn try_init_tracing() -> Result<(), SetGlobalDefaultError> {
+pub(crate) fn try_init_tracing(testing: bool) -> Result<(), SetGlobalDefaultError> {
```

Comment:
A bool parameter here isn't very clear. Can we instead split this into:
`init_test_tracing` and `try_init_tracing`?

I'd consider also making `try_init_layered_tracing` take a level instead, since we probably want to duplicate this in the `try_init_wasm_tracing` version